sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java

```diff
@@ -179,7 +180,8 @@
                 .flatMap((PhoneNumbersSearchAvailablePhoneNumbersResponse response) -> {
                     pollingContext.setData("operationId", response.getDeserializedHeaders().getOperationId());
                     pollingContext.setData("searchId", response.getDeserializedHeaders().getSearchId());
-                    return client.getOperationAsync(pollingContext.getData("operationId"));
+                    Mono<PhoneNumberOperation> operation = getOperation(pollingContext.getData("operationId"));
+                    return operation;
```

Comment:
nit: any particular reason why this is being assigned to local variable before being returned?

Reply:
Oh grreat catch! I was doing some testing and totally forgot to revert this. Thank you, I'll fix

Reply:
i figured but didn't want to assume :)